modules/pel/peltool/peltool.py

```diff
@@ -267,7 +267,82 @@ def parseAndWriteOutput(file: str, output_dir: str, config: Config,
             print(f"No PEL parsed for {file}: {e}")
 
 
+def parsePELSummary(stream: DataStream, config: Config, critSysTerm:bool = False):
+    """
+    Parses and summarizes data from a PEL stream.
+    Returns: Event ID (eid) and summary of PEL data.
+            If no PEL is parsed or an error occurs, empty strings are returned.
+    """
+    out = OrderedDict()
+    ret, ph = generatePH(stream, out)
+    if ret is False:
+        return "", ""
+    eid = ph.pLID
+    ret, uh = generateUH(stream, ph.creatorID, out)
+    if ret is False:
+        return "", ""
+    if not uh.isServiceable():
+        return "", ""
+    
+    summary = OrderedDict()
+    for _ in range(2, ph.sectionCount):
+        sectionID, sectionLen, versionID, subType, componentID = parseHeader(stream)
+        section_json = OrderedDict()
+        sectionFun(stream, section_json, sectionID, sectionLen,
+                   versionID, subType, componentID, ph.creatorID, config)
+        if sectionID == SectionID.primarySRC.value:
+            summary["SRC"] = section_json["Primary SRC"]["Reference Code"]
+            if "Error Details" in section_json["Primary SRC"] :
+                summary["Message"] = section_json["Primary SRC"]["Error Details"]["Message"]
+            break
+    summary["PLID"] = ph.pLID
+    summary["CreatorID"] = out["Private Header"]["Creator Subsystem"]
+    summary["Subsystem"] = out["User Header"]["Subsystem"]
+    summary["Commit Time"] = ph.commitTime
+    summary["Sev"] = out["User Header"]["Event Severity"]
+    summary["CompID"] = out["Private Header"]["Created by"]
+    return eid, summary
+
+def extractAndSummarizePEL(file: str, config: Config,
+                        delete_after_parsing: bool):
```

Comment:
`delete_after_parsing` is unused, are you defining unwanted arguments and passing through all functions from `main` function.

Reply:
Currently not using these extra arguments, will be useful for future feature implementation.

Reply:
Its ok for now but, we should push into relevant patch..Consider this practise in future.